src/components/scenes/CoinRankingScene.tsx

```diff
@@ -242,8 +239,7 @@ const CoinRankingComponent = (props: Props) => {
           </View>
           <DividerLine marginRem={[0, 0, 0, 1]} />
           <View style={{ ...undoInsetStyle, marginTop: 0 }}>
-            <AnimatedFlashList
-              estimatedItemSize={theme.rem(3.75)}
+            <Animated.FlatList
```

Comment:
Interesting. Does this work with the gesture handler? Not sure what happens if we require both reanimated and gesture-handler support.

Reply:
It appears to work. Isn't reanimated and gesture handler both by software mansion? I figured they'd both be using the same gesture handling efficiencies, but maybe that assumption is wrong?

Reply:
If it works, it works.